src/Microsoft.SqlTools.ServiceLayer/EditData/EditDataService.cs

```diff
@@ -86,6 +86,7 @@ public void InitializeService(ServiceHost serviceHost)
             serviceHost.SetRequestHandler(EditSubsetRequest.Type, HandleSubsetRequest, true);
             serviceHost.SetRequestHandler(EditUpdateCellRequest.Type, HandleUpdateCellRequest, true);
             serviceHost.SetRequestHandler(EditCommitRequest.Type, HandleCommitRequest, true);
+            serviceHost.SetRequestHandler(EditScriptRequest.Type, HandleEditScriptRequest, true);
```

Comment:
While you're here, can you update all these SetRequestHandler() calls to name the magic constant?

e.g. `isParallelProcessingSupported: true`